### PR TITLE
Ariadne/job migration

### PR DESCRIPTION
**Pull request by mgill25**
Migration of the Dataproc jobs from Ariadne (part of `christos-test` cluster), along with the appropriate settings, connections etc.

This PR will be tested at `https://ned8a20ea93a4374b-tp.appspot.com/admin/`

### CODE REVIEW

File: dags/etl-jobs.py [hunk 1]
```diff
@@ -0,0 +1,108 @@
...
+
+    delete_dataproc_cluster = DataprocClusterDeleteOperator(
+        task_id='delete_dataproc_cluster',
+        cluster_name='datascience-cluster-ariadne',
+        project_id='fromatob-legacy',
+        # Setting trigger_rule to ALL_DONE causes the cluster to be deleted
+        # even if the Dataproc job fails.
+        trigger_rule=trigger_rule.TriggerRule.ALL_DONE)
+
+    create_dataproc_cluster >> conversions >> pageviews >> search >> app_metrics >> service_fee >> database_connector >> delete_dataproc_cluster
```
Comment (cadicallegari) [inline dags/etl-jobs.py]: Are there any jobs that can run in parallel?

File: hack/setup.sh [hunk 2]
```diff
@@ -22,6 +22,10 @@ airflow connections -a --conn_type http --conn_id fb_app_id --conn_password "foo
 airflow connections -a --conn_type http --conn_id fb_access_token --conn_password "foo"
 airflow connections -a --conn_type http --conn_id fb_app_secret --conn_password "foo"
 
+airflow connections -a --conn_type google_cloud_platform --conn_id google_cloud_datastore_default --conn_extra '{"extra__google_cloud_platform__project": "a2b-insights"}'
```
Comment (cadicallegari) [inline hack/setup.sh]: Is there any automation to create these connections in prod?

Comment (mgill25) [reply hack/setup.sh]: Not for now because then we get into the same problem of secrets (connections will store things like API keys and passwords).

Lets leave it manual for now and try to improve in future.

File: dags/etl-jobs.py (hunk 1 above)
Comment (mgill25) [reply dags/etl-jobs.py]: Some do run in parallel (mediaspend and mediaspend30d). The DAG makes it fairly clear. We can try to optimize the pre-existing ones later also.

Comment (cadicallegari) [reply dags/etl-jobs.py]: Maybe I used the wrong word =(
Let me rephrase that
Some tasks of this dag (which are dataproc jobs) can run in parallel?

File: hack/setup.sh (hunk 2 above)
Comment (cadicallegari) [reply hack/setup.sh]: Do we have an automate way to create a composer env?

Comment (mgill25) [reply hack/setup.sh]: Terraform

File: dags/etl-jobs.py (hunk 1 above)
Comment (mgill25) [reply dags/etl-jobs.py]: That is up to dataproc itself. I do not know enough about these jobs to tell you. Maybe @xalexakis can answer better.
